package.json

```diff
@@ -15,7 +15,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "cy:open": "react-scripts start & cypress open"
   },
```

Comment:
We all decided to install the yarn cypress script?

Reply:
To just open cypress on its own without the server?

Reply:
yarn add cypress -D start-server-and-test